tools/codegen/model.py

```diff
@@ -695,6 +822,16 @@ def get_kernel(self, g: Union[NativeFunction, NativeFunctionsGroup]) -> Optional
         if isinstance(g, NativeFunction):
             f = g
         elif isinstance(g, NativeFunctionsGroup):
+            # If you're fetching the kernel for an operator that is using ufunc
+            # codegen, there will be no entry in the dispatch table; instead, there
+            # is an implicit kernel that will be generated by ufunc codegen.
+            # TODO: Maybe this should already be precomputed so the if statement
+            # here in BackendIndex isn't necessary.
+            if g.out.ufunc_inner_loop and is_ufunc_dispatch_key(self.dispatch_key):
```

Comment:
One nice thing about pre-computing it is for debugging - I remember a few times where I had pdb open and I wanted to check what kernels a given backend had, and you can do that by printing the values in self.index without worrying about them not all being populated. (this seems minor and not worth blocking the PR on though)

Reply:
It looks like this can't be done conveniently. I'll add a comment about this...

Reply:
OK I worked harder to do it and I think I nailed it.